include/mega/command.h

```diff
@@ -1205,6 +1205,44 @@ class MEGA_API CommandFolderLinkInfo: public Command
     CommandFolderLinkInfo(MegaClient*, handle);
 };
 
+class MEGA_API CommandSyncPut : public Command
```

Comment:
Dismissable picky proposal: throughout all this PR, change `Sync` by `HeartbeatedSync`, or something like that to differentiate this `Sync` concept from the actual Sync engine stuff. Sync related app callbacks are also sync*** which might become confusing overtime. In any case: specs talk about Syncs whereas it could be other stuff like Camera Uploads, I guess it's a little late for a full rebranding, but we still might want to consider it within the SDK ;-)

Reply:
Yes, I'll apply a refactoring

Reply:
Regarding refactoring these new Commands (an also param names of the heartbeats, like `syncId` or `syncName`), I agree that not always it's a `Sync`, but sometimes a CU or likely anything else in the future.
A different naming should be assigned, and I vote for renaming `sync` to `backup` wherever it's appropriate. In the end, it is for the backup centre: clients register backups, send backup-heartbeats, etc. So... *backup* everywhere :)